tests/apichecks/crds_test.go

```diff
@@ -581,7 +589,7 @@ func findFieldsNotCoveredByTests(t *testing.T, shouldVisitCRD func(crd *apiexten
 
 					// Only report an error if neither external nor name is set
 					if !hasExternal && !hasName {
-						errs = append(errs, fmt.Sprintf("[missing_field] crd=%s version=%v: field %q is not set; neither 'external' nor 'name' are set", crd.Name, version.Name, fieldPath))
+						untestedRefErrs = append(untestedRefErrs, fmt.Sprintf("[untested_ref_field] crd=%s version=%v: field %q is not set, neither 'external' nor 'name' is set in unstructured objects.", crd.Name, version.Name, fieldPath))
```

Comment:
I don't dislike this but here's what I am thinking:

* let's first mark the new field, something like [missing_field][untested_ref_field]
* let's move the file (this will show up as a git mv)
* then we can drop the missing_field tag/ replace it everywhere.

You can do this in separate commits too! It will just make it so much easier for us to review!